demos/sslecho/main.c

```diff
@@ -0,0 +1,356 @@
+/*
+    Copyright 20xx-20yy The OpenSSL Project Authors. All Rights Reserved.
+
+    Licensed under the Apache License 2.0 (the "License").  You may not use
+    this file except in compliance with the License.  You can obtain a copy
+    in the file LICENSE in the source distribution or at
+    https://www.openssl.org/source/license.html
+ */
+
+#include <stdio.h>
+#include <unistd.h>
+#include <stdbool.h>
+#include <string.h>
+#include <sys/socket.h>
+#include <arpa/inet.h>
+#include <openssl/ssl.h>
+#include <openssl/err.h>
+
+/**
+ * This program implements an SSL echo client/server.
+ *
+ * Spin up the server, then the client. The client sends lines from keyboard
+ * input to the server, which then echos the lines back to the client.
+ *
+ * Revision History
+ *
+ * 11-Dec-2021      RCW
+ *
+ * 1.0.0    Initial version; working with 2 instances on one machine.
+ *
+ * 1.0.1    Binding server to INADDR_ANY (instead of 127.0.0.1).
+ *          Client needs server ip on command line.
+ *          Using ERR_print_errors_fp() for all SSL errors.
+ *          Added kill command to terminate server.
+ *
+ * 1.0.2    Replaced 'server-' & 'client-' pem files with default
+ *          generated cert.pem and key.pem.
+ */
+static const char*      revision = "1.0.2";
+
+static const int        server_port = 4433;
```

Comment:
Use standard OpenSSL style. So no extra spaces. Space between "char" and "*":

```` C
static const char *revision = "1.0.2";
static const int server_port = 4433;
````

Similarly for other variable declarations below